platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/NativeMapView.java

```diff
@@ -42,6 +45,8 @@
     // Used for callbacks
     private MapView mMapView;
 
+    private final float pixelRatio;
+
```

Comment:
Considered that, but the native map view already has to know about density on both sides anyway.